composer.json

```
@@ -1,8 +1,15 @@
 {
-  "name": "c3-cloudfront-clear-cache",
-  "version": "0.0.0",
-  "type": "wordpress-plugin",
-  "require": {
-    "php": ">=5.5"
-  }
+    "name": "humanmade/c3-cloudfront-clear-cache",
+    "description": "Modernised fork of a simple plugin that clear all cloudfront cache if you publish posts.",
```

Minor changes suggested,
```suggestion
    "description": "Modernised fork of a simple plugin that clears all CloudFront caches if you publish posts.",
```

Aside: is it "if", or "when"?